Specs/Scene/TileMapServiceImageryProviderSpec.js

```diff
@@ -400,11 +402,11 @@ describe("Scene/TileMapServiceImageryProvider", function () {
     const provider = new TileMapServiceImageryProvider({
       url: "http://server.invalid?query=1",
     });
+    provider.readyPromise.catch(() => {});
```

Comment:
Here and throughout- We should never blindly catch an error in the specs, as it could (and probably is in many cases) covering up an underlying issue. If the test is expected to throw an error

1) There should be an expect statement ensuring the type or message of the error thrown is the intended one
2) There should be a `fail()` stipulation to catch in case the promise resolves instead of rejecting
3) The test name should be specific enough to mention that throwing an error is expected, ie. "throws if format is unsupported" or "rejects on invalid url"

Reply:
Added these where appropriate.